types/src/block.rs

```diff
@@ -0,0 +1,331 @@
+use tendermint::block::{Commit, CommitSig, Header, Id};
+use tendermint::signature::SIGNATURE_LENGTH;
+use tendermint::vote;
+use tendermint::{chain, Hash, Vote};
+
+use crate::consts::{genesis::MAX_CHAIN_ID_LEN, version};
+use crate::{Error, Result, ValidateBasic, ValidationError, ValidationResult};
+
+pub const GENESIS_HEIGHT: u64 = 1;
```

Comment:
remove the `pub` or move this in `consts.rs`

Reply:
as it's only used now and not vendored from celestia I just removed the pub